src/DDoSX/AclGeoIpClient.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace UKFast\SDK\DDoSX;
+
+use UKFast\SDK\Client as BaseClient;
+
+class AclGeoIpClient extends BaseClient
+{
+    protected $basePath = 'ddosx/';
+
+    /**
+     * Delete an ACL GeoIp Rule for a domain
+     * @param $domainName
+     * @param $geoIpRuleId
+     * @return bool
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function destroyRule($domainName, $geoIpRuleId)
```

Comment:
in the rest of the sdk methods, we simply use `destroy` to delete an entity, so I'd do the same here to stay consistent

```suggestion
    public function destroy($domainName, $geoIpRuleId)
```

Reply:
since the acl geo ip endpoint allows `mode` configuration as well, it makes sense to have this split up